configs/env-example

```diff
@@ -11,7 +11,7 @@ BUILD_ID=0
 API_PORT=3000
 APP_PORT=8080
 SWAGGER_ENABLED=true
-APPLICATION_ARCHIVE_DAYS=43
+APPLICATION_ARCHIVE_DAYS=365
```

Comment:
Can we also adjust E2E tests to use new value

![image](https://github.com/user-attachments/assets/c3ab1d52-a978-40be-8490-138009b15281)

This is as per following technical AC.

![image](https://github.com/user-attachments/assets/468eff8f-05ad-40e1-a81c-c29033c8a063)

Reply:
Updated the test. Thanks!